packages/e2e/src/batch-request.test.ts

```diff
@@ -0,0 +1,48 @@
+import { afterAll, describe, expect, it } from 'vitest'
+
+import networks from './networks'
+
+describe('Batch request', async () => {
+  const acala = await networks.acala()
+  const { chain, wsClient } = acala
```

Comment:
```suggestion
  const { chain, url } = acala
  const webSocket = new WebSocket(url)
```

maybe we can do this?

Reply:
yea, we can do this, but I feel it's cleaner to set it up in testing, and we may reuse it in future

Reply:
if we keep then it needs a proper error handling. Current code will not resolve if connect fails. So better remove it

Reply:
yes, we need to handle connect fail, but we still need to connect right? no matter where we put this code

Reply:
yes, put it on the batch test